### DISCUSSION TITLE
Are there the documents about how the dims/coordinate should be interpreted?

### DISCUSSION THREAD

**Discussion by arafune** [pydata/xarray#9295] [General] [answered]
Dear Sir:
Concerning the issue in holoviews:

https://github.com/holoviz/holoviews/issues/6327

I'm searching for the appropriate documents about  how the dims/coordinate should be interpreted. 
Especially I would like to know whether the coordinate object is expected the ordered object
Unfortunately, I have not found it.

in 

https://docs.xarray.dev/en/stable/user-guide/data-structures.html

> coords: a list or dictionary of coordinates. If a list, it should be a list of tuples where the first element is the dimension name and the second element is the corresponding coordinate array_like object.  

While I don't feel that coords is expected they are ordered, I'm not 100% sure about it.
The point is the current python dict object is ordered. So question is whether the coords object expects this ``ordered`` feature of the dict object or not.

Could you let me know if you have the corresponding description?
If not, please let me know the answer of the above question.

Thank you for your help in advance.

**Comment by keewis** [comment, marked as answer]
only the dimension names on variables are expected to be ordered, everything else is not. In particular, the order of coordinates and variables only affects the representation of the dataset (e.g. `repr(ds)`).
